src/fontra/client/core/ui-utils.js

```diff
@@ -94,13 +94,19 @@ function didReorder(a, b) {
   return false;
 }
 
+export function checkboxWithoutLabel(controller, key, options) {
```

Comment:
Please get rid of this function, and just call `labeledCheckbox()` with `null` as the first argument (*not* `false`).

Unless you really prefer a separate function, but then please call it `unlabeledCheckbox` instead. And same: please use `null` instead of `false`.

(I've seen you do this before (pass "false" to say "no value"), but that's never nice or conventional. I'd even say it's incorrect. If value is optional, it should be replacable with `null` or `undefined`, but never with `false`.)